### PR TITLE
[Geospatial data] Display attributions for the layers on the map

### PR DESCRIPTION
**Pull request by clementprdhomme**
This PR displays the sources of the active layers as attributions on the map.

## Acceptance criteria

- At the bottom of the module, there is the following text: “Powered by:” followed by the list of sources of the active layers
  - The Resource Watch and Global Forest Watch sources have a link to their homepage

## Tracking

[ORC-615](https://vizzuality.atlassian.net/browse/ORC-615)

[ORC-615]: https://vizzuality.atlassian.net/browse/ORC-615?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: client/src/containers/map/attributions/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+        let content = name;
+        if (name === 'RW' || name === 'ResourceWatch' || name === 'Resource Watch') {
+          content = `<a href="https://resourcewatch.org/" rel="noopener noreferrer">${name}</a>`;
+        } else if (
+          name === 'GFW' ||
+          name === 'GlobalForestWatch' ||
+          name === 'Global Forest Watch'
+        ) {
+          content = `<a href="https://www.globalforestwatch.org/" rel="noopener noreferrer">${name}</a>`;
```
Comment (Bluesmile82) [inline client/src/containers/map/attributions/index.tsx:52]: I think this should have target="_blank" as they are external pages. Also for the RW link

**Comment by Bluesmile82** [comment]
image
This is an edge case but there is a point in which you can't see the Mapbox part anymore. We should allow the user to see it or have a tooltip.

**Comment by clementprdhomme** [comment]
@Bluesmile82 I had trouble to find a solution to fit the attributions on smaller screens. The main problem is that I don't control the attributions related to the basemap. I can only pass HTML for the custom attributions.

I thought that we could alternatively render our own attributions component instead of using the one from React Map GL/Mapbox but in that case I don't know how to receive the basemap's attributions programmatically.

Do you have any suggestion for me?

**Comment by Bluesmile82** [comment]
> Preview

I think one option would be to use the current static attribution that they have. Maybe it will change in the future but probably won't be a big change. 

Another one is remove the line-clamp. On mobile it wont be a problem as we don't have the sidebar and if its too big it automatically shrinks to the compact version (info button). And on desktop it will be max two lines so maybe we have to move the legend a bit up

**Review by Bluesmile82 — APPROVED** [review]
Good enough! Thanks!

**Pull request merged**